### PR TITLE
Update Db.php

### PR DESCRIPTION
**Pull request by adamoki**
In a case I have NULL in data value it is converted to string 'NULL' which is not acceptable, so I've fixed it to accept NULL.

### CODE REVIEW

**Comment by zf2timo** [comment]
@adamoki Please add a unit test which covers the behavior.

File: src/Writer/Db.php [hunk 1]
```diff
@@ -163,7 +163,7 @@ protected function mapEventIntoColumn(array $event, array $columnMap = null)
             if (is_array($value)) {
                 foreach ($value as $key => $subvalue) {
                     if (isset($columnMap[$name][$key])) {
-                        if (is_scalar($subvalue)) {
+                        if (is_scalar($subvalue) || is_null($subvalue)) {
```
Comment (Ocramius) [inline src/Writer/Db.php:166]: Please use `null === $subvalue` instead of `is_null()`

Comment (adamoki) [inline src/Writer/Db.php:166]: Ok, there is a little bit difference in speed comparsion.

**Comment by weierophinney** [comment]
Closing due to lack of updates.

**Pull request closed without merging**